### ISSUE TITLE
First item of right Y-Axis legend is hard to click

### ISSUE THREAD

**Issue by grmat** [grafana/grafana#27590]
When using right Y-axis, the first (rightmost) legend label is hard to click. Only a small fraction on the left side of the label is actually clickable. Almost impossible to hit on touch devices.

I assume this is due to the negative margin of the parent item (`<div class="view" ...>`)

![recording](https://user-images.githubusercontent.com/12658837/93191834-46273480-f745-11ea-8eb8-3c9921a5bebd.gif)

**Comment by lukewing** [comment]
How can I recreate this in my local environment?

**Comment by grafanabot** [comment]
Thanks for creating this issue! We think it's missing some basic information. 

Follow the issue template and add additional information that will help us replicate the problem. 
For data visualization issues: 
- Query results from the inspect drawer (data tab & query inspector)
- Panel settings can be extracted in the panel inspect drawer JSON tab

For dashboard related issues: 
- Dashboard JSON can be found in the dashboard settings JSON model view

For authentication, provisioning and alerting issues, Grafana server logs are useful. 

Happy graphing!

**Comment by grafanabot** [comment]
We've closed this issue since it needs more information and hasn't had any activity recently. We can re-open it after you you add more information. To avoid having your issue closed in the future, please read our [CONTRIBUTING](https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md) guidelines.

Happy graphing!
